restore-postgres-backup/README.md

```diff
@@ -0,0 +1,34 @@
+# Restore Postgres Backup
+
+Restore an Azure Postgresql server database from a previous backup that is stored in an Azure storage account
+Mainly designed to be used in DR and DR test procedures
+
+## Inputs
+- `storage-account`: Name of the Azure atorage account that contains the backup (Required)
+- `resource-group`: Azure resource group of the storage account (Required)
+- `app-name`: Snyk token for image vulnerability scanning (Required)
+- `azure-credentials`: A JSON string containing service principle credentials (Required)
+- `environment`: Application environment that containes the database restore target (Required)
+- `backup-file`: Name of the source backup file that is being restored (Required)
+- `compression`: Is the backup file compressed? true or false (Default:false)
```

Comment:
Can this be updated now that the input is removed?

Reply:
removed